app/src/main/java/at/aau/edu/wizards/gameModel/GameModelCpu.kt

```diff
@@ -2,6 +2,7 @@ package at.aau.edu.wizards.gameModel
 
 import kotlin.random.Random
 
+// FIXME why does cpu need a gamemodelplayer in most methods
```

Comment:
Because it needs to know which player to get the guess, get next played card, etc. from.

You could however switch it to an id and not pass an player object, I can fix that.

Reply:
Instead, redesign the CPU. Should the player object be part of the CPU? Lets discuss this next week

Reply:
switched to id